app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -268,7 +269,7 @@ private void initViews() {
     }
 
     private void setUpMenu() {
-        final View sheet = getLayoutInflater().inflate(R.layout.bottom_sheet_main_menu, null);
+        final View sheet = getLayoutInflater().inflate(R.layout.bottom_sheet_main_menu, (ViewGroup) null);
```

Comment:
This seems unnecessary.

Reply:
There's a lint warning about inflate should pass a view parent, not null. In our case, we inflate the view for a Dialog to setView, there's no way we can have the reference of it. This bypass the lint warning. 
However, this is only a warning not a fatal error of lint. maybe I should just skip this block.

Reply:
It's a warning and from code quality perspective I don't think adding this is enhancing it. (Actually a new warning popups: Unnecessary cast) so I think the better approach would be ignoring this lint case by case when we're 100% sure we don't have a parent. Before doing that I think we can leave the warning there as a reminder.